sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenExchangeServiceCallbackMock.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.appcenter.storage;
+
+import com.microsoft.appcenter.storage.client.TokenExchange;
+import com.microsoft.appcenter.storage.models.TokenResult;
+
+public class TokenExchangeServiceCallbackMock extends TokenExchange.TokenExchangeServiceCallback {

```

Comment:
What is this for? You should be able to mock everything that you want.

Reply:
hmmm.. tried to stub the class... but this class is an abstract class called by a service callback in a service callback... got a little hard to write the test case, thus directly implement it... :see_no_evil: